src/installation/guides/fde.md

```diff
@@ -126,11 +126,15 @@ Before we enter the chroot to finish up configuration, we do the actual install.
 keys](../../xbps/troubleshooting/common-issues.md#verifying-rsa-keys) for the
 packages you are installing.
 
+It should be noted that the next command need to be issued in accordance with
+your chosen libc implementation, if musl libc, the link should be directed to:
+`current/musl`, if GNU libc, it should be directed to: `current`.
```

Comment:
This isn't complete, aarch64 uses `current/aarch64`

See the `chroot.md` guide, if you could link to it directly from the FDE guide instead of repeating the information here, I think that'd be ideal - ideally this guide would contain only the FDE specific parts, with links to the relevant parts of the chroot one for the rest.